contracts/stake-cw20/src/state.rs

```diff
@@ -3,12 +3,14 @@ use serde::{Deserialize, Serialize};
 
 use cosmwasm_std::{Addr, Uint128};
 use cw_controllers::Claims;
+use cw_controllers::Hooks;
 use cw_storage_plus::{Item, SnapshotItem, SnapshotMap, Strategy};
 use cw_utils::Duration;
 
 #[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
 pub struct Config {
-    pub admin: Option<Addr>,
+    pub owner: Option<Addr>,
+    pub manager: Option<Addr>,
```

Comment:
should we add comments describing the differences in behavior for these two roles (and their use cases).

Reply:
Agree

Reply:
+1

Reply:
:heavy_check_mark: